client/my-sites/upgrades/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -0,0 +1,152 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import analyticsMixin from 'lib/mixins/analytics';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import support from 'lib/url/support';
+
+const learnMoreLink = <a href={ support.COMPLETING_GOOGLE_APPS_SIGNUP } target="_blank" />,
+	strong = <strong />;
+
+const PendingGappsTosNotice = React.createClass( {
+	mixins: [ analyticsMixin( 'domainManagement', 'googleApps' ) ],
+
+	propTypes: {
+		site: React.PropTypes.oneOfType( [
```

Comment:
It seems we are not using `site` for anything else than tracking, do you think it'd be a good idea to just pass the slug and call it `siteSlug`?